### PR TITLE
Clarity update

### PR DESCRIPTION
**Pull request by aaronedell**
Fixed typos, rewrote sections, improved clarity.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 1]
```diff
@@ -81,63 +81,75 @@ cid-cmd deploy
 
 {{% /expand%}}
 
-## Option 2: CloudFormation deployment
+## Option 2: All-in-one CloudFormation deployment (10 min)
 
-### All-in-one CloudFormation deployment (10 min)
+This option does not require any pre-requisites other than the CUR having been created and available in an S3 bucket.
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:86]: Also QS (Enterprise Support and SICE)

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 2]
```diff
@@ -81,63 +81,75 @@ cid-cmd deploy
...
 4. Review **Readme Sectons** parameter to confirm prerequisites before specifying the other parameters. You must answer 'yes' to all.
-5. Set **QuicksightUserName** parameter
-To validate your QuickSight complete the tasks below:
+5. Copy and paste your **QuicksightUserName** into the parameter text box
+To find your QuickSight username:
 	- Open a new tab or window and navigate to the **QuickSight** console
 	- Find your username in the top right navigation bar
 ![Images/cf_dash_qs_2.png](/Cost/200_Cloud_Intelligence/Images/cf_dash_qs_2.png?classes=lab_picture_small)
 
+1. Update your **CURBuketPath** with the S3 path where your **partitions of CUR data** starts. If you have just one CUR in this bucket, put the path to the folder where the year folders are. For example `s3://cid-accountid-shared/cur/accountid/cid/cid/`. If you have more than on CUR in the bucket from multiple accounts, you will put an S3 path that just takes you to the account level of one of your CURs (doesn't matter which one), for example `s3://cid-accountid-shared/cur/177834886075/`.
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:108]: Do we want to keep '177834886075' i would rather replace with 123456789012

File: config.toml [hunk 3]
```diff
@@ -83,3 +83,4 @@ weight = 10
 name = "<i class='fas fa-bookmark'></i>&nbsp;&nbsp;Amazon Free Tier"
 url = "https://aws.amazon.com/free/"
 weight = 20
+theme = "learn"
```
Comment (iakov-aws) [inline config.toml:86]: Not sure if it is intended to be here

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 4]
```diff
@@ -210,6 +210,83 @@ Follow the below steps to remove the colum. Be aware this will mean if you do ad
...
+
+Edit your Glue crawler to refresh the metadata from the partitions by following [these steps.](https://aws.amazon.com/premiumsupport/knowledge-center/athena-hive-partition-schema-mismatch/) 
+
+{{% /expand%}}
+
+##  I am getting the following error when attempting to update the dashboards using the cid-cmd tool; An error occurred (InvalidParameterValueException) when calling the UpdateDashboard operation: Given placeholders [cudos_cur] are not part of template. 
+
+{{%expand "Click here to expand answer" %}}
+
+Rename your QuickSight dataset for your raw CUR to customer_all, click save and publish, and let it refresh. Try updating the dashboards again.
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:247]: Not sure if it is frequent case

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 5]
```diff
@@ -210,6 +210,83 @@ Follow the below steps to remove the colum. Be aware this will mean if you do ad
...
+
+{{%expand "Click here to expand answer" %}}
+
+For a KMS encrypted bucket, KEY and ROLE policies need to be modified to allow you to use the KMS key that encrypts the S3 bucket. Additional security features like SCP can also be applied in your Organization. For troubleshooting IAM permissions you can use CloudTrail and open support tickets with cloud support engineers. Here is a sample KMS key policy that should be added in cases where the CUR bucket is KMS encrypted. The policy allows QuickSight and Glue roles to decrypt the data encrypted with that KMS key.
+
+IMPORTANT: This statement is to be added to an existing kms key policy and not to replace it, the key could have other policies in place used by other services, like key administrators policy.
+replace account with account number
+replace CRAWLER-ROLE with cur glue crawler IAM role (can be found in crawler config/Service role)
+
+{ "Sid": "Allow Quicksight and Glue", "Effect": "Allow", "Principal": { "AWS": [ "arn:aws:iam::{account}:role/service-role/aws-quicksight-service-role-v0", "arn:aws:iam::{account}:role/{CRAWLER-ROLE}" ] }, "Action": "kms:Decrypt", "Resource": "*" }
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:285]: Athena and QS service roles must have the same access ?

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:108]: also for this kind of path `s3://cid-accountid-shared/cur/accountid/cid/cid/` CURisAggregated must be = no

So we need to repharse it 

```
1. Set CUR **CURBuketPath**  and **CURisAggregated** flag

Option2: Did you installed CUR with CFN on the previous step? 
**CURBuketPath**  - put the path to the folder where account ids are. `s3://cid-1234567890123-shared/cur/` 
**CURisAggregated** = 'yes'

Option1: Do you have just one account and you installed CUR manually?
**CURBuketPath**  - put the path to the folder where the year folders are. For example `s3://cid-accountid-shared/prefix/name/name/` (double check this path, you must see year=xxxx partitions there)
**CURisAggregated** = 'no'
```

Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:108]: good catch

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md (hunk 1 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:86]: fixed, also added a note in prereqs about purchasing spice capacity

File: config.toml (hunk 3 above)
Comment (aaronedell) [reply config.toml:86]: that is the theme the WA uses but I was having some trouble with my dev environment and re-added the theme

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md (hunk 4 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:247]: you're right - its probably not frequent. happened once I think

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md (hunk 5 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:285]: I dunno - this was from Timur, is it wrong?

Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:269]: thats ok, QS is there, and Athena editor uses the access of a user, so this should be fine. 

just let's format a bit for more readability
```
{
    "Sid": "Allow Quicksight and Glue",
    "Action": "kms:Decrypt",
    "Effect": "Allow",
    "Principal":
    {
        "AWS":
        [
            "arn:aws:iam::{account}:role/service-role/aws-quicksight-service-role-v0",
            "arn:aws:iam::{account}:role/{CRAWLER-ROLE}"
        ]
    },
    "Resource": "arn:aws:kms:{region}:{account}:key/*"
}
```

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 6]
```diff
@@ -81,63 +81,77 @@ cid-cmd deploy
...
 	- Open a new tab or window and navigate to the **QuickSight** console
 	- Find your username in the top right navigation bar
 ![Images/cf_dash_qs_2.png](/Cost/200_Cloud_Intelligence/Images/cf_dash_qs_2.png?classes=lab_picture_small)
 
+1. Update your **CURBuketPath** and set the **CURisAggregated** flag;
+   1. If you used the CFN Template in the CUR setup process earlier in this lab ([option 1)](/cost/200_labs/200_cloud_intelligence/cost-usage-report-dashboards/dashboards/1a_cur_setup/)then;
+      1. **CURBucketPath** needs to be the s3 path to the folder in your CUR bucket where account IDs are. For example `s3://cid-1234567890123-shared/cur/`
+      2. **CURisAggregated** = `yes`
+   2. If you did *not* use the CFN automated CUR setup process in the prior lab step and have just one CUR you setup manually then;
+      1. **CURBucketPath** needs to be the s3 path to the folder in your CUR bucket where the year folders are. For example `s3://cid-accountid-shared/prefix/name/name/` (double check this path, you must see year=xxxx partitions in there)
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:113]: missing

2. **CURisAggregated** = `no`

**Review by iakov-aws — APPROVED** [review]

**Pull request merged**